### ISSUE TITLE
使用fun-asr-nano转录异常，使用Paraformer Large模型正常。

### ISSUE THREAD

**Issue by mvpzhangkai** [Quantatirsk/funasr-api#3]
按照您的教程，使用您的容器镜像，容器化部署后，使用fun-asr-nano模型的时候，后台日志提示如下。而且识别出来是异常的乱码，使用另一个Paraformer Large模型转录识别是正常的，请问是什么原因？

GPU是 A100 40g.

2025-12-24 14:37:51,589 - [worker-10] - app.services.asr.engine - INFO - 识别分段 1/7: 0.00s - 50.10s
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:151645 for open-end generation.
2025-12-24 14:38:04,852 - [worker-10] - app.services.asr.engine - INFO - 识别分段 2/7: 50.10s - 98.74s
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:151645 for open-end generation.
2025-12-24 14:38:17,948 - [worker-10] - app.services.asr.engine - INFO - 识别分段 3/7: 98.74s - 150.40s
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:151645 for open-end generation.
2025-12-24 14:38:30,963 - [worker-10] - app.services.asr.engine - INFO - 识别分段 4/7: 150.40s - 200.71s
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:151645 for open-end generation.
2025-12-24 14:38:44,115 - [worker-10] - app.services.asr.engine - INFO - 识别分段 5/7: 200.71s - 249.37s
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:151645 for open-end generation.
2025-12-24 14:38:57,139 - [worker-10] - app.services.asr.engine - INFO - 识别分段 6/7: 249.37s - 268.69s
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:151645 for open-end generation.
2025-12-24 14:39:10,445 - [worker-10] - app.services.asr.engine - INFO - 识别分段 7/7: 268.69s - 272.96s
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:151645 for open-end generation.
2025-12-24 14:39:23,602 - [worker-10] - app.services.asr.engine - INFO - 长音频识别完成，共 7 个有效分段，总字符数: 16916
2025-12-24 14:39:23,603 - [worker-10] - app.api.v1.asr - INFO - [291ff93d9ad65434c7c66b9f30a91ab6] 识别完成，共 7 个分段，总字符: 16916
INFO:     192.168.112.3:36452 - "POST /stream/v1/asr?appkey=hJPT4E3B%2ANWk&model_id=fun-asr-nano&vocabulary_id=%E9%98%BF%E9%87%8C%E5%B7%B4%E5%B7%B4%2020%20%E8%85%BE%E8%AE%AF%2015 HTTP/1.1" 200 OK

**Comment by Quantatirsk** [comment]
> 按照您的教程，使用您的容器镜像，容器化部署后，使用fun-asr-nano模型的时候，后台日志提示如下。而且识别出来是异常的乱码，使用另一个Paraformer Large模型转录识别是正常的，请问是什么原因？
> 
> GPU是 A100 40g.
> 
> 2025-12-24 14:37:51,589 - [worker-10] - app.services.asr.engine - INFO - 识别分段 1/7: 0.00s - 50.10s The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results. Setting `pad_token_id` to `eos_token_id`:151645 for open-end generation. 2025-12-24 14:38:04,852 - [worker-10] - app.services.asr.engine - INFO - 识别分段 2/7: 50.10s - 98.74s The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results. Setting `pad_token_id` to `eos_token_id`:151645 for open-end generation. 2025-12-24 14:38:17,948 - [worker-10] - app.services.asr.engine - INFO - 识别分段 3/7: 98.74s - 150.40s The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results. Setting `pad_token_id` to `eos_token_id`:151645 for open-end generation. 2025-12-24 14:38:30,963 - [worker-10] - app.services.asr.engine - INFO - 识别分段 4/7: 150.40s - 200.71s The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results. Setting `pad_token_id` to `eos_token_id`:151645 for open-end generation. 2025-12-24 14:38:44,115 - [worker-10] - app.services.asr.engine - INFO - 识别分段 5/7: 200.71s - 249.37s The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results. Setting `pad_token_id` to `eos_token_id`:151645 for open-end generation. 2025-12-24 14:38:57,139 - [worker-10] - app.services.asr.engine - INFO - 识别分段 6/7: 249.37s - 268.69s The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results. Setting `pad_token_id` to `eos_token_id`:151645 for open-end generation. 2025-12-24 14:39:10,445 - [worker-10] - app.services.asr.engine - INFO - 识别分段 7/7: 268.69s - 272.96s The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results. Setting `pad_token_id` to `eos_token_id`:151645 for open-end generation. 2025-12-24 14:39:23,602 - [worker-10] - app.services.asr.engine - INFO - 长音频识别完成，共 7 个有效分段，总字符数: 16916 2025-12-24 14:39:23,603 - [worker-10] - app.api.v1.asr - INFO - [291ff93d9ad65434c7c66b9f30a91ab6] 识别完成，共 7 个分段，总字符: 16916 INFO: 192.168.112.3:36452 - "POST /stream/v1/asr?appkey=hJPT4E3B%2ANWk&model_id=fun-asr-nano&vocabulary_id=%E9%98%BF%E9%87%8C%E5%B7%B4%E5%B7%B4%2020%20%E8%85%BE%E8%AE%AF%2015 HTTP/1.1" 200 OK

回头复测下，我是 4090 的卡没问题，2080ti 22g 不行，不知道是不是显卡架构的原因

**Comment by mvpzhangkai** [comment]
> > 按照您的教程，使用您的容器镜像，容器化部署后，使用fun-asr-nano模型的时候，后台日志提示如下。而且识别出来是异常的乱码，使用另一个Paraformer Large模型转录识别是正常的，请问是什么原因？
> > GPU是 A100 40g.
> > 2025-12-24 14:37:51,589 - [worker-10] - app.services.asr.engine - INFO - 识别分段 1/7: 0.00s - 50.10s The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results. Setting `pad_token_id` to `eos_token_id`:151645 for open-end generation. 2025-12-24 14:38:04,852 - [worker-10] - app.services.asr.engine - INFO - 识别分段 2/7: 50.10s - 98.74s The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results. Setting `pad_token_id` to `eos_token_id`:151645 for open-end generation. 2025-12-24 14:38:17,948 - [worker-10] - app.services.asr.engine - INFO - 识别分段 3/7: 98.74s - 150.40s The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results. Setting `pad_token_id` to `eos_token_id`:151645 for open-end generation. 2025-12-24 14:38:30,963 - [worker-10] - app.services.asr.engine - INFO - 识别分段 4/7: 150.40s - 200.71s The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results. Setting `pad_token_id` to `eos_token_id`:151645 for open-end generation. 2025-12-24 14:38:44,115 - [worker-10] - app.services.asr.engine - INFO - 识别分段 5/7: 200.71s - 249.37s The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results. Setting `pad_token_id` to `eos_token_id`:151645 for open-end generation. 2025-12-24 14:38:57,139 - [worker-10] - app.services.asr.engine - INFO - 识别分段 6/7: 249.37s - 268.69s The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results. Setting `pad_token_id` to `eos_token_id`:151645 for open-end generation. 2025-12-24 14:39:10,445 - [worker-10] - app.services.asr.engine - INFO - 识别分段 7/7: 268.69s - 272.96s The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results. Setting `pad_token_id` to `eos_token_id`:151645 for open-end generation. 2025-12-24 14:39:23,602 - [worker-10] - app.services.asr.engine - INFO - 长音频识别完成，共 7 个有效分段，总字符数: 16916 2025-12-24 14:39:23,603 - [worker-10] - app.api.v1.asr - INFO - [291ff93d9ad65434c7c66b9f30a91ab6] 识别完成，共 7 个分段，总字符: 16916 INFO: 192.168.112.3:36452 - "POST /stream/v1/asr?appkey=hJPT4E3B%2ANWk&model_id=fun-asr-nano&vocabulary_id=%E9%98%BF%E9%87%8C%E5%B7%B4%E5%B7%B4%2020%20%E8%85%BE%E8%AE%AF%2015 HTTP/1.1" 200 OK
> 
> 回头复测下，我是 4090 的卡没问题，2080ti 22g 不行，不知道是不是显卡架构的原因


我看官方的模型文件有更新，更换了最新的模型文件和model.py，ctc.py，正常了，谢谢。
